src/common/components/Label/Label.theme.ts

```diff
@@ -0,0 +1,33 @@
+import { type VariantProps, tv } from "tailwind-variants";
+
+export type LabelVariants = VariantProps<typeof labelTheme>;
+
+export const labelTheme = tv(
+  {
+    slots: {
+      label: [
+        "bg-transparent",
+        "outline-none",
+        "placeholder:text-text-em-low",
+        "flex-1",
+      ],
+      wrapper: ["flex", "items-center", "gap-2", "pl-1"],
+    },
+    variants: {
+      size: {
+        md: {
+          label: ["text-sm"],
+        },
+        sm: {
+          label: ["text-xs"],
+        },
+      },
+      error: {
+        true: {
+          wrapper: ["text-red-600"],
```

Comment:
Feel like we should probably abstract the color to  `/themes` as it will be commonly used throughout the app for error/danger messages. Not sure if it's too premature though!

Reply:
great point! addressed at [`d7b9d5b`](https://github.com/AfterClass-io/afterclass.io-v2/pull/27/commits/d7b9d5b6b15412333388bc404d8783a9da1f3a3d)